addons/web/__manifest__.py

```diff
@@ -62,6 +62,8 @@
             'web/static/src/legacy/scss/ui.scss',
 
             'web/static/src/polyfills/clipboard.js',
+            'web/static/src/polyfills/object.js',
+            'web/static/src/polyfills/array.js',
```

Comment:
Please don't add these to assets backend

Reply:
Sorry, my bad, I missed that part in AAB's message. I removed them 👍🏻